serde_v8/src/serializable.rs

```diff
@@ -20,3 +22,96 @@ impl<T: serde::Serialize> Serializable for T {
     crate::to_v8(scope, self)
   }
 }
+
+/// SerializablePkg exists to provide a fast path for op returns,
+/// allowing them to avoid boxing primtives (ints/floats/bool/unit/...)
+pub enum SerializablePkg {
```

Comment:
The name of this enum is questionable, what does `Pkg` refer to?

Reply:
We could call it `BoxedSerializable` and just say that it's functionally identical to `Box<dyn serde_v8::Serializable>` but more efficient for primitives (since many ops return primitives, e.g: ResourceIds, etc...)

Reply:
Let's keep the current name for now